### PR TITLE
add fix for missing _sqlite problem

### PR DESCRIPTION
**Pull request by mjdominus**
When I ran `./python -m coverage` per the instructions, using current development `cpython` (version 3.14.0a0) I got the following strange-seeming error:

    $ python -m coverage
    Traceback (most recent call last):
      File "/home/mjd/pkg/cpython/Lib/runpy.py", line 189, in _run_module_as_main
        mod_name, mod_spec, code = _get_module_details(mod_name, _Error)
                                   ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^
      File "/home/mjd/pkg/cpython/Lib/runpy.py", line 148, in _get_module_details
        return _get_module_details(pkg_main_name, error)
      File "/home/mjd/pkg/cpython/Lib/runpy.py", line 112, in _get_module_details
        __import__(pkg_name)
        ~~~~~~~~~~^^^^^^^^^^
      File "/home/mjd/.venv/cpython/lib/python3.14/site-packages/coverage/__init__.py", line 24, in <module>
        from coverage.control import (
        ...<2 lines>...
        )
      File "/home/mjd/.venv/cpython/lib/python3.14/site-packages/coverage/control.py", line 29, in <module>
        from coverage.collector import Collector, HAS_CTRACER
      File "/home/mjd/.venv/cpython/lib/python3.14/site-packages/coverage/collector.py", line 19, in <module>
        from coverage.data import CoverageData
      File "/home/mjd/.venv/cpython/lib/python3.14/site-packages/coverage/data.py", line 24, in <module>
        from coverage.sqldata import CoverageData
      File "/home/mjd/.venv/cpython/lib/python3.14/site-packages/coverage/sqldata.py", line 16, in <module>
        import sqlite3
      File "/home/mjd/pkg/cpython/Lib/sqlite3/__init__.py", line 57, in <module>
        from sqlite3.dbapi2 import *
      File "/home/mjd/pkg/cpython/Lib/sqlite3/dbapi2.py", line 27, in <module>
        from _sqlite3 import *
    ModuleNotFoundError: No module named '_sqlite3'

The suggestion at https://stackoverflow.com/a/24449632/1277934 fixed this for me.


<!-- readthedocs-preview cpython-devguide start -->
----
📚 Documentation preview 📚: https://cpython-devguide--1325.org.readthedocs.build/

<!-- readthedocs-preview cpython-devguide end -->

### CODE REVIEW

**Comment by mjdominus** [comment]
@encukou

File: testing/coverage.rst [hunk 1]
```diff
@@ -158,6 +158,12 @@ such as by venv)::
 
     ./python -m coverage
 
+(If you get the error
+
+    ImportError: No module named _sqlite3
+
+rerun ``configure`` with the ``--enable-loadable-sqlite-extensions`` flag.)
```
[the remark this answers is not in the document]
Comment (mjdominus) [reply testing/coverage.rst:165]: @erlend-aasland But it _did_ help me, so your diagnosis can't be exactly correct.

[the remark this answers is not in the document]
Comment (erlend-aasland) [reply testing/coverage.rst:165]: There has to be other changes in your environment; take a look at the code in `configure.ac` for more details.

[the remark this answers is not in the document]
Comment (erlend-aasland) [reply testing/coverage.rst:165]: Hint: take a look at the `have_sqlite3` and `have_supported_sqlite3` variables in `configure.ac`.

[the remark this answers is not in the document]
Comment (mjdominus) [reply testing/coverage.rst:165]: OK, investigating exactly what I did in what order probably isn't worth doing for this change, which would have had marginal value anyway, so I'm closing the PR.

Thanks for your assistance.

[the remark this answers is not in the document]
Comment (erlend-aasland) [reply testing/coverage.rst:165]: Ok, thanks for creating a PR, though! I still think we could improve the devguide regarding this. I'll create an issue.

**Pull request closed without merging**